app/models/saved_claim/pension.rb

```diff
@@ -1,11 +1,15 @@
 # frozen_string_literal: true
 class SavedClaim::Pension < SavedClaim
   FORM = '21P-527EZ'
+  CONFIRMATION = 'PEN'
+  ATTACHMENT_KEYS = [:files].freeze
+  PERSISTENT_CLASS = PersistentAttachment::PensionBurial
+
   def regional_office
     PensionBurial::ProcessingOffice.address_for(open_struct_form.veteranAddress.postalCode)
   end
 
-  def confirmation_number
-    "V-PEN-#{id}#{guid[0..6]}".upcase
+  def attachment_keys
+    self.class::ATTACHMENT_KEYS
```

Comment:
Ok, so now this feels a little like it's passing a jankiness threshold. ;) Why not implement it as an instance method in the first place? Or just use the class variable every time you need to reference it?